<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showCodes</class>
 <widget class="QWidget" name="showCodes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>353</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <widget class="QWidget" name="page_lb">
        <widget class="QLabel" name="projectName_Lb">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>NewProject</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_le">
        <widget class="QLineEdit" name="projectName_Le">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_Pb">
       <property name="text">
        <string>保存到</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icon/save.png</normaloff>:/icon/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="run_Pb">
       <property name="text">
        <string>运行</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icon/run.png</normaloff>:/icon/run.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fileTree_Pb">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icon/folderOpen.png</normaloff>:/icon/folderOpen.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
